source/magnifier/fullscreenMagnifier.py

```diff
@@ -0,0 +1,463 @@
+		log.info(
+			f"Starting magnifier with zoom level {self.zoomLevel} and filter {self.filterType} and fullscreen mode {self.fullscreenMode}",
+		)
+		# Initialize Magnification API if not already initialized
+		try:
+			magnification.MagInitialize()
+			log.debug("Magnification API initialized")
+		except Exception as e:
+			# Already initialized or failed - continue anyway
+			log.debug(f"MagInitialize result: {e}")
+
+		if self.isActive:
+			self._applyFilter()
+		self._startTimer(self._updateMagnifier)
+
+	def _doUpdate(self):
+		"""
+		Perform the actual update of the magnifier
+		"""
+		# Calculate new position based on focus mode
+		x, y = self._getCoordinatesForMode(self.currentCoordinates)
+		# Always save screen position for mode continuity
+		self.lastScreenPosition = (x, y)
+
+		if self.lastFocusedObject == FocusType.NVDA:
+			try:
+				from .config import shouldKeepMouseCentered
+			except ImportError:
+				log.error("Failed to import shouldKeepMouseCentered from magnifier.config")
+			else:
+				if shouldKeepMouseCentered():
+					self.moveMouseToScreen()
+		# Apply transformation
+		self._fullscreenMagnifier(x, y)
+
+	def _stopMagnifier(self) -> None:
+		"""
+		Stop the Fullscreen magnifier using windows DLL
+		"""
+		super()._stopMagnifier()
+		try:
+			# Get MagSetFullscreenTransform function from magnification API
+			MagSetFullscreenTransform = self._getMagnificationApi()
+			# Reset fullscreen magnifier: 1.0 zoom, 0,0 position
+			MagSetFullscreenTransform(ctypes.c_float(1.0), ctypes.c_int(0), ctypes.c_int(0))
+			# Reset color effect to normal (identity matrix)
+			magnification.MagSetFullscreenColorEffect(FilterMatrix.NORMAL.value)
+		except Exception as e:
+			log.info(f"Error resetting magnification: {e}")
+
+		# Uninitialize Magnification API
+		try:
+			magnification.MagUninitialize()
+			log.debug("Magnification API uninitialized")
+		except Exception as e:
+			log.debug(f"MagUninitialize result: {e}")
+
+	def _applyFilter(self) -> None:
+		"""
+		Apply the current color filter to the fullscreen magnifier
+		"""
+		try:
+			if self.filterType == Filter.NORMAL:
+				magnification.MagSetFullscreenColorEffect(FilterMatrix.NORMAL.value)
+			elif self.filterType == Filter.GRAYSCALE:
+				magnification.MagSetFullscreenColorEffect(FilterMatrix.GRAYSCALE.value)
+			elif self.filterType == Filter.INVERTED:
+				magnification.MagSetFullscreenColorEffect(FilterMatrix.INVERTED.value)
+			else:
+				log.info(f"Unknown color filter: {self.filterType}")
+		except Exception as e:
+			log.error(f"Failed to apply filter: {e}")
+
+	def _getMagnificationApi(self):
+		"""
+		Get Windows Magnification API function
+		"""
+		MagSetFullscreenTransform = ctypes.windll.magnification.MagSetFullscreenTransform
+		MagSetFullscreenTransform.restype = ctypes.wintypes.BOOL
+		MagSetFullscreenTransform.argtypes = [ctypes.c_float, ctypes.c_int, ctypes.c_int]
+		return MagSetFullscreenTransform
+
+	def _fullscreenMagnifier(self, x: int, y: int) -> None:
```

Comment:
use coordinates here: 
```suggestion
	def _fullscreenMagnifier(self, coordinates: Coordinates) -> None:
```